examples/ex9.cpp

```diff
@@ -135,6 +135,8 @@ class FE_Evolution : public TimeDependentOperator
    virtual void ImplicitSolve(const double dt, const Vector &x, Vector &k);
 
    virtual ~FE_Evolution();
+
+   MemoryClass GetMemoryClass() const { return Device::GetMemoryClass(); }
```

Comment:
Is this function used anywhere?

Reply:
This function is not used anymore (when I started this branch months ago it was), and [I have now removed it](94e07ba735dd370d3dbd89d4bc54b2872c6793c9).